kubernetes/update-stack.sh

```diff
@@ -355,7 +343,6 @@ EOF
     echo "setting influxdb..."
     # it is assumed that the commands below may fail.
     set +e
-    setup_influxdb
     TOKEN_NAME="waggle-read-write-bucket"
     # NOTE(sean) there have been nodes with multiple tokens named 'waggle-read-write-bucket', so we simply accept the first match.
     WAGGLE_INFLUXDB_TOKEN=$(kubectl exec svc/wes-node-influxdb -- influx auth ls | awk -v name="${TOKEN_NAME}" '$2 ~ name {print $3; exit}')
```

Comment:
Do you plan on just setting this token ahead of time and passing it as an env var to InfluxDB or would you like to keep the existing approach for now?

Reply:
I would like to keep this `waggle-read-write-bucket` token and separate it from the admin token. The reasons are,
- it's already working and `wes-node-influxdb-loader` is using that token,
- it may be the case where plugins would use a token to query influxDB. if possible, I want to avoid giving plugins the admin token